src/components/Button/index.view.tsx

```diff
@@ -5,11 +5,22 @@ export interface ButtonProps {
   text: string;
   link: string;
   label: string;
+  className?: string;
 }
 
-const Button: React.FC<ButtonProps> = ({ text, link, label }: ButtonProps) => {
+const Button: React.FC<ButtonProps> = ({
+  text,
+  link,
+  label,
+  className,
+}: ButtonProps) => {
```

Comment:
In order to contextualize the Button props, I have added this optional `className` prop to the `ButtonProps`. This will allow you to contextualize the button based on where on the site you are. @tim-nguyen-cs, this is an example of how powerful `className` is in terms of contextualizing each component.

Reply:
Ahh, that makes sense! I love the look - we can contextualize globally by passing in a class name within its props or locally when we define the element itself!